fv3core/README.md

```diff
@@ -194,6 +196,54 @@ Add `PULL=False` to build from scratch without running `docker pull`:
 PULL=False make build
 ```
 
+### Bare-Metal
+
+Pace requires gcc > 9.2, cuda, MPI, and Python 3.8 on your system in order to run bare-metal.
+
+You will also need the headers of the boost libraries in your `path` (boost itself does not need to be installed).
+
+```shell
+cd $BOOST_LOCATION
+wget https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.gz
+tar -xzf boost_1_79_0.tar.gz
+mkdir -p boost_1_79_0/include
+mv boost_1_79_0/boost boost_1_79_0/include/.
```

Comment:
If we had a simpler `Dockerfile` I feel like a link to that part of the `Dockerfile` would be easier to keep in sync. WDYT?